frontend/integration-tests/tests/kubevirt/template.wizard.scenario.ts

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-undef, max-nested-callbacks */
+
+import { execSync } from 'child_process';
+import { browser, by, ExpectedConditions as until } from 'protractor';
+import { OrderedMap } from 'immutable';
+
+import { appHost, testName } from '../../protractor.conf';
+import { resourceRowsPresent, filterForName, deleteRow, createItemButton, isLoaded, errorMessage } from '../../views/crud.view';
+import { removeLeakedResources } from './utils';
+import { testNAD } from './mocks';
+import * as vmView from '../../views/kubevirt/vm.view';
+
+const VM_BOOTUP_TIMEOUT = 90000;
+const PAGE_LOAD_TIMEOUT = 5000;
+const VM_WIZARD_LOAD_TIMEOUT = 10000;
+
+describe('Kubevirt create VM template using wizard', () => {
+  const leakedResources = new Set<string>();
+  const vmName = `vm-${testName}`;
+  const tmplName = `template-${testName}`;
+  const operatingSystem = 'Red Hat Enterprise Linux 7.6';
+  const flavor = 'small';
+  const workloadProfile = 'generic';
+  const sourceURL = 'https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img';
+  const sourceContainer = 'kubevirt/cirros-registry-disk-demo:latest';
+  const pxeInterface = 'nic1';
+  const provisionMethods = OrderedMap<string, (provisionSource: string) => void>()
```

Comment:
Seems we are reusing these. Consider moving these to mocks.ts.

Reply:
> Hello, nice effort!
> Could you please update the scenario to reflect changes that are being introduced in #212 ?

OK, I will update it once 212 is merged.